modules/installation-guide/partials/proc_building-an-offline-devfile-registry-image.adoc

```diff
@@ -0,0 +1,40 @@
+// Used in:
+// * installation-guide/installing-che-in-a-restricted-environment/
+// * installation-guide/upgrading-che-using-the-cli-management-tool-in-restricted-environment/
+
+[id="building-an-offline-devfile-registry-image_{context}"]
+= Building an offline devfile registry image
+
+This section describes how to build an offline devfile registry image.
+Starting workspaces without relying on resources from the outside Internet requires building this image.
+The image contains all sample projects referenced in devfiles as `zip` files.
+
+.Prerequisites:
+
+* NodeJS 12.x
```

Comment:
This is incorrect, and appears to be a bug in the current docs:

https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-in-a-restricted-environment/#building-offline-devfile-registry-image_che

You do not need NodeJS or yarn to build an offline devfile registry. It's needed for the plugin registry, so this just looks like a copy/paste mistake.

All that's needed is podman/docker installed. All the bits about NodeJS/yarn can be removed.

Reply:
Good!